docs/introduction/faq-and-common-issues.md

```diff
@@ -148,3 +149,16 @@ See [Outgoing emails](https://github.com/djfarrelly/MailDev#outgoing-email) in t
 
 ## Can I see what is really happening in the Codeception acceptance tests when using Docker?
 Yes, you can! Check [the quick guide](/docs/introduction/running-acceptance-tests.md#how-to-watch-what-is-going-on-in-the-selenium-browser).
+
+## Why is there a faked PHP 7.2 platform in the Composer config?
+As a general rule, packages and libraries that depend on PHP 7.2 will work as expected even on PHP 7.3 (any higher 7.x version), but not vice versa.
+This is the maintainers of PHP focus on backward-compatibility (even if there were [some incompatible changes](https://www.php.net/manual/en/migration73.incompatible.php) introduced in PHP 7.3, in practice it doesn't cause issues).
+
+Using [the `config.platform.php` option](https://getcomposer.org/doc/06-config.md#platform) in `composer.json` allows us to force Composer to install such dependencies, that work for all supported versions of PHP by Shopsys Framework.
+These dependencies are locked during each release of SSFW so users that install it can download exact versions of all libraries and tools that were tested and proved working.
+This helps to eliminate unforeseen issues during installation.
+See [Composer docs](https://getcomposer.org/doc/01-basic-usage.md#installing-with-composer-lock) for more details on version locking.
+
+Without this forced platform version, you could encounter issues when working on your project with developers that use a different version of PHP.
```

Comment:
```suggestion
Without this forced platform version, you could encounter issues when working on your project with developers that use a different version of PHP. E.g. your `composer.lock` could contain dependencies that not all developers can install.
```

Reply:
There is no linebreak on the rendered page on GitHub:

![image](https://user-images.githubusercontent.com/10008612/63698399-0bdbb400-c81f-11e9-9799-4f797ae93d6f.png)

Keeping sentences on different lines is better for maintaining clean diffs. We should probably add that to the [Guidelines for Writing Documentation](https://github.com/shopsys/shopsys/blob/v8.0.0/docs/contributing/guidelines-for-writing-documentation.md)

Reply:
Got it, it is OK. And how about using `e.g.` instead `For example` - I'm not sure if it is OK at the beggining of sentece.